subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -88,7 +88,7 @@ public void onOutput(OutputEvent event) {
                     phaseHasMoreProgress(startEvent);
                 } else if (startEvent.getBuildOperationCategory() == BuildOperationCategory.CONFIGURE_PROJECT && currentPhase == Phase.Configuring) {
                     currentPhaseChildren.add(startEvent.getProgressOperationId());
-                } else if (startEvent.getBuildOperationCategory() == BuildOperationCategory.TASK && currentPhase == Phase.Executing) {
+                } else if ((startEvent.getBuildOperationCategory() == BuildOperationCategory.TASK || startEvent.getBuildOperationCategory() == BuildOperationCategory.TRANSFORM) && currentPhase == Phase.Executing) {
```

Comment:
Just the condition in this line is 185 characters long. That's too much, please refactor to something more readable.

Reply:
I moved the necessary logic into the enum.